### DISCUSSION TITLE
似乎在google patents會無法使用此擴充功能？

### DISCUSSION THREAD

**Discussion by amy776262002** [nextai-translator/nextai-translator#124] [Q&A]
真的是非常實用的擴充功能！感謝您

不過不太確定是否是我自己電腦的問題，
但在安裝此擴充功能後，chrome好像會無法打開https://patents.google.com/ 這個頁面？

**Comment by vincentckuo** [comment]
> 真的是非常實用的擴充功能！感謝您
> 不過不太確定是否是我自己電腦的問題， 但在安裝此擴充功能後，chrome好像會無法打開https://patents.google.com/ 這個頁面？

True, I have the same problem so I turn this extension off.
